modules/gdscript/editor/script_templates/CharacterBody3D/basic_movement.gd

```diff
@@ -5,29 +5,29 @@ extends _BASE_
 
 const SPEED = 5.0
 const JUMP_VELOCITY = 4.5
+const FRICTION = 10.0
 
-# Get the gravity from the project settings to be synced with RigidBody nodes.
+# Get the gravity from the project settings, to allow syncing with RigidBody3D nodes.
 var gravity: float = ProjectSettings.get_setting("physics/3d/default_gravity")
 
 
 func _physics_process(delta: float) -> void:
 	# Add the gravity.
-	if not is_on_floor():
-		velocity.y -= gravity * delta
+	velocity.y -= gravity * delta
```

Comment:
> Apply gravity always, not just when in the air.

But why?

Reply:
While this is useful for sliding on slopes (and I don't remember if it's in the documentation that you have to apply gravity) it's a special use case and I think it'd cause confusion and I suspect also unnecessary work for the engine (not familiar with optimizations being done in that case but I doubt it is free)

Reply:
> But why?

Because that's how gravity works. If you stop downward acceleration when you're on the ground, you're not on the ground, you're hovering above it. `move_and_slide()` adjusts the velocity appropriately.